lib/travis/api/app/endpoint/authorization.rb

```diff
@@ -104,7 +104,7 @@ class Authorization < Endpoint
       #
       # * **redirect_uri**: URI to redirect to after handshake.
       get '/handshake/?:provider?' do
-        method = Travis::Features.enabled_for_all?(:vcs_login) ? :vcs_handshake : :handshake
+        method = !org? ? :vcs_handshake : :handshake
```

Comment:
Please do not use negation
Instead of it you can use com? or just 
org? ? :handshake : :vcs_handshake

Reply:
Also please add some specs

Reply:
I mean add spec for org and not org - we should make sure that it will be disabled for org